src/Symfony/Component/Form/Tests/AbstractLayoutTest.php

```diff
@@ -528,6 +528,25 @@ public function testSingleChoice()
         );
     }
 
+    public function testSelectWithSizeBiggerThanOneCanBeRequired()
+    {
+        $form = $this->factory->createNamed('name', 'choice', '&a', array(
```

Comment:
remember `'Symfony\Component\Form\Extension\Core\Type\ChoiceType'` must be used in 2.8